configure.ac

```diff
@@ -29,7 +29,7 @@ PKG_CHECK_MODULES([DBUS], [dbus-1])
 PKG_CHECK_MODULES([GIO], [gio-unix-2.0])
 PKG_CHECK_MODULES([GLIB], [glib-2.0])
 PKG_CHECK_MODULES([GOBJECT], [gobject-2.0])
-PKG_CHECK_MODULES([SAPI],[sapi])
+PKG_CHECK_MODULES([SAPI],[sapi < 2.0.0])
```

Comment:
Consider to leave some warning messages to 3rd-party developers when sapi >= 2.0.0 is found
(instead of such a fatal error with no detailed messages, which might disturb anyone trying to build tabrmd with an unstable version of libsapi)